sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/SessionRecords/SecretClientLiveTests/CrudWithExtendedProps.json

```diff
@@ -0,0 +1,207 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://pakrym-keyvault.vault.azure.net/secrets/1622923008?api-version=7.0",
```

Comment:
should we change the recording to target the `ssnetvault` vault, instead of your personal one? just making sure that when people run the live test locally the settings/permission for the vault are the same and content is known

Reply:
Yes, we should but we would need to switch `ssnetvault` to soft-delete mode first.

Reply:
I don't think it particularly matters which vault the recordings target.

Reply:
We could change it later to which ever vault we are going to use for nightly runs to make sure all environments are consistent.